templates/account.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block body %}
+<h1>Create an Account:</h1>
+<form action="/user/add" method="POST">
+    {% with errors = get_flashed_messages(category_filter=["account_err"]) %}
+        {% if errors %}
+            {%- for msg in errors %}
+                <font color='red' id='msg'>{{ msg }}</font>
+            {% endfor -%}
+        {% endif %}  
+    {% endwith %}
```

Comment:
This message flashing should be in `base.html` -- that way it will be available on _all_ pages.